webapp/config/packages/security.yaml

```diff
@@ -3,10 +3,10 @@
 security:
     role_hierarchy:
         ROLE_JURY: [ROLE_CLARIFICATION_RW, ROLE_API, ROLE_API_READER, ROLE_API_SOURCE_READER]
-        ROLE_ADMIN: [ROLE_JURY, ROLE_JUDGEHOST, ROLE_API_WRITER]
+        ROLE_ADMIN: [ROLE_JURY, ROLE_JUDGEHOST, ROLE_API_WRITER,
+                     ROLE_API_PROBLEM_CHANGE, ROLE_API_CONTEST_CHANGE]
```

Comment:
I think `editor` would be a better suffix than `change`. Also, is `ROLE_API_CONTEST_CHANGE` a superset of privileges over `ROLE_API_PROBLEM_CHANGE`?

Reply:
No, a contest is a different thing from a problem as we allow problems without a contest.